src/webportal/src/app/layout/alerts.jsx

```diff
@@ -0,0 +1,120 @@
+import c from 'classnames';
+import {Panel, List, mergeStyleSets, getFocusStyle, getTheme, initializeIcons, PanelType, Stack, StackItem} from 'office-ui-fabric-react';
+import React, {useCallback, useState, useEffect} from 'react';
+
+import {initTheme} from '../components/theme';
+import config from '../config/webportal.config';
+
+initTheme();
+initializeIcons();
```

Comment:
The initialization functions and missed polyfills (`corejs`, `fetch`...) should be in the root file `layout.js` instead of a component.

Reply:
`initializeIcons` should only be called once but every page rely on `layout.component.js`.
Maybe we need to refactor the init function calls

Reply:
Remove these init function. Currently, it works. We can refactor the init function call when we have time.